tokumx/pyproject.toml

```diff
@@ -10,6 +10,7 @@ name = "datadog-tokumx"
 description = "The TokuMX check"
 readme = "README.md"
 license = "BSD-3-Clause"
+requires-python = "==2.7.*"
 keywords = [
```

Comment:
Should we also remove https://github.com/DataDog/integrations-core/pull/11555/files#diff-4ef0a0419e00f133c0e210c76f4528dab6ba0e55ccf0c16f734266f4efe37b6bR29?